### PR TITLE
Add cURL example, and instructions on linked accounts

### PR DESCRIPTION
**Pull request by heydonovan**
Fixes https://github.com/objectrocket/documentation/issues/60.

Also, added instructions on creating a password should the user sign-in with their Rackspace account.

### CODE REVIEW

**Comment by evanfreed** [comment]
LGTM

**Comment by SethSanchez** [comment]
You should strip out part of your token or something, I think the term is obfuscation

**Comment by SethSanchez** [comment]
Ah, it's probably fine, it's a token, it'll expire

**Comment by heydonovan** [comment]
The token isn't real. I just generated 81 characters :p

File: source/api_v2_getting_started.rst [hunk 1]
```diff
@@ -47,37 +52,48 @@ One important note. The **token** is what will be used for the **X-Auth-Token**
...
+   Vary: Accept-Encoding
+   X-RateLimit-Limit: 100
+   X-RateLimit-Remaining: 99
+   X-RateLimit-Reset: 1449083646
+   X-Request-Id: a07dbb8ac2aa4dee91f1825e69ab6a5d
+   
+   {
+       "data": [
+           {
+               "api_key": "4aa8a323d92d4253bd02c0992d621560",
```
Comment (DavidWittman) [inline source/api_v2_getting_started.rst]: You may want to sanitize this. :camel:

File: source/api_v2_getting_started.rst [hunk 2]
```diff
@@ -47,37 +52,48 @@ One important note. The **token** is what will be used for the **X-Auth-Token**
...
-                "service": "mongodb",
-                "ssl_connect_string": "syd-mongos0.objectrocket.com:45023",
-                "type": "mongodb_sharded",
-                "version": "2.4.6",
-                "zone": "AP-Sydney"
-            }
-        ]
-    }
+   $ curl 'https://sjc-api.objectrocket.com/v2/instances/' --header 'X-Auth-Token: xLQiuUySDpyDjgGlaBiLbQmrdPYlKJETEIHLisZVxXcFDCsXKNQdohbsdErLsTKBWAmdYCgLKDjhNxYfE'
+   {"data": [{"service": "mongodb", "connect_string": "syd-mongos0.objectrocket.com:35023", "name": "MongoDB123", "zone": "AP-Sydney", "created": "2015-04-29 09:53:18", "ssl_connect_string": "syd-mongos0.objectrocket.com:45023", "encrypted": false, "storage_engine": "mmapv1", "ssl_service_net_connect_string": "syd-sn-mongos0.objectrocket.com:45023", "service_net_connect_string": "syd-sn-mongos0.objectrocket.com:35023", "compressor": "none", "version": "2.4.6", "plan": 5, "caster": null, "api_key": "4aa8a323d92d4253bd02c0992d621560", "type": "mongodb_sharded", "id": "55410c7f5b335278490a5be8"}]}
```
Comment (DavidWittman) [inline source/api_v2_getting_started.rst]: This too

**Comment by heydonovan** [comment]
Can I get another plus one? :) /cc @evanfreed @DavidWittman 

The `token` isn't real, and the instance has been nuked, so the `api_key` isn't an issue.

**Comment by DavidWittman** [comment]
LGTM

**Comment by SethSanchez** [comment]
Das coo

**Pull request merged**